apache/uima-uimacpp
# Test cases for Aggregate CAS Multiplier

**What's in the PR**
 New tests to check aggregate engine's ability to handle CAS Multipliers. For now these tests will fail if ran
- [AggregateCASMultiplier.xml](https://github.com/mac-op/uima-uimacpp/blob/enhancement/capability-tests/src/test/data/descriptors/AggregateCASMultiplier.xml) features a simple CAS Multiplier.
- Another test will feature a CAS Multiplier that combines CASes instead of dividing, as suggested by @DrDub.

**How to test **
* Change [this](https://github.com/mac-op/uima-uimacpp/blob/a7ebb4d5bc86a3ce46c5666a67c315492c58a2ca/src/test/src/test_engine.cpp#L586) and run `make check`, which will fail.

**Automatic testing**

**Documentation**

**Organizational**
      <sub><sup>Only dependencies under [approved licenses](http://www.apache.org/legal/resolved.html#category-a) are allowed. LICENSE and NOTICE files in the respective modules where dependencies have been added as well as in the project root have been updated.</sup></sub>
